api/android/build-android-lib.sh

```diff
@@ -298,9 +298,10 @@ if [[ $enable_nnfw == "yes" ]]; then
         tar -zxf external/onert-ext-$nnfw_ver-android-aarch64.tar.gz -C external/nnfw
     fi
 
-    # Remove duplicated, unnecessary files (c++shared and tensorflowlite_jni)
+    # Remove duplicated, unnecessary files (c++shared and tensorflow-lite)
     rm -f external/nnfw/lib/libc++_shared.so
     rm -f external/nnfw/lib/libtensorflowlite_jni.so
+    rm -f external/nnfw/lib/libneuralnetworks.so
```

Comment:
is this tf-lite nnapi?

Reply:
https://github.com/nnstreamer/nnstreamer/issues/2952#issuecomment-747780800

No it's Android's library. But I guess NNFW has packaged it for some internal reason

Reply:
NNFW team internally uses this library for test (tf-lite nnapi).